apm-agent-api/src/main/java/co/elastic/apm/api/TransactionImpl.java

```diff
@@ -164,4 +165,16 @@ public Transaction setOutcome(Outcome outcome) {
         doSetOutcome(outcome);
         return this;
     }
+
+    @Nonnull
+    @Override
+    public Transaction setDestinationAddress(@Nullable String address, int port) {
+        return this;
+    }
+
+    @Nonnull
+    @Override
+    public Transaction setDestinationService(@Nullable String resource) {
```

Comment:
Deprecate this method so that users know setting the destination only works for spans.

Reply:
added

Reply:
added